src/HPRT/toolkit/views.py

```diff
@@ -330,43 +331,72 @@ class HopkinsPart2DetailView(LoginRequiredMixin, DetailView):
     template_name = 'hp2_detail.html'
 
 
-def make_pdf(canvas, info, questions):
-    textobject = canvas.beginText()
-    textobject.setTextOrigin(inch, 10*inch)
-    textobject.setFillColorRGB(0.4,0,1)
-    textobject.setFont("Helvetica-Oblique", 60)
-    textobject.textLines("HPRT")
-    textobject.setFont("Times-Bold", 13)
-    textobject.textLines("")
-    textobject.setFillColor(colors.black)
-    textobject.textLines("Patient Name: " + info["patient"])
-    textobject.textLines("Doctor Administered: Dr. " + info["doctor"])
-    textobject.textLines("Date Administered: " + info["date"])
-
-    para_questions = []
+# def make_pdf(canvas, info, questions):
```

Comment:
If yours works fine, you should just delete all of these comments.  If we need to roll back the changes for whatever reason, they're stored here on Github and we can use that for reference if need be.